custom_mutators/rust/example/src/lib.rs

```diff
@@ -43,4 +43,31 @@ impl CustomMutator for OwnBufferExampleMutator {
     }
 }
 
+struct FallibleMutatorExample;
+
+impl FallibleCustomMutator for FallibleMutatorExample {
+    type Error = i32;
+
+    fn init(seed: c_uint) -> Result<Self, i32> {
+        Ok(Self)
+    }
+
+    fn fuzz<'b, 's: 'b>(
+        &'s mut self,
+        buffer: &'b mut [u8],
+        add_buff: Option<&[u8]>,
+        max_size: usize,
+    ) -> Result<Option<&'b [u8]>, i32> {
+        Err(1)
+    }
+
+    fn handle_err(err: i32) {
+        if std::env::var("DEBUG").is_ok() {
```

Comment:
This should just be default behavior inside the bindings, people can handle their own errors if they really need to :)

Reply:
done!